apps/table.html

```diff
@@ -255,6 +255,7 @@ <h3>Steps for uploading.</h3>
                                                 </div> </div>
                                                 <a title="Upload from google drive" href="#" id='gdriveUpload' onclick="googlePickerStart()"><i style="font-size: 1.2em;" class="fab fa-google-drive"></i></a>
                                                 <a href="#" id='urlswitch' onclick="$('#urlInput').val(''); switchToUrl();" > <span style="position: absolute; right:2em;" > <i class="fas fa-link"></i> Upload from URL</span></a>
+												<a title="Upload from Dropbox" href="#" id='dropboxUpload' onclick="dropboxChooserStart()"><i style="font-size: 1.2em;" class="fab fa-dropbox"></i></a>
```

Comment:
I think you forgot to re-add the config and Dropbox api to this file. In the other PR you also added:

```
<script type="text/javascript" src="https://www.dropbox.com/static/api/2/dropins.js" id="dropboxjs"></script>
<script src="./loader/config.js"></script>
```

Reply:
Whoopsie. My bad. Added in next commit :)